scripts/variantstore/wdl/GvsUtils.wdl

```diff
@@ -93,12 +96,20 @@ task SplitIntervals {
     set -e
     export GATK_LOCAL_JAR=~{default="/root/gatk.jar" gatk_override}
 
+    WEIGHT_BED_JAVE_ARG="~{using_weighted_beds}"
```

Comment:
looks like a misspelling, unused below

Reply:
and that's why you shouldn't quickly rename any variables AFTER a successful run.  Good catch